ratis-server/src/main/java/org/apache/ratis/server/impl/PeerConfiguration.java

```diff
@@ -63,10 +65,6 @@
     return Collections.unmodifiableMap(map);
   }
 
-  PeerConfiguration(Iterable<RaftPeer> peers) {
-    this(peers, Collections.emptyList());
-  }
-
```

Comment:
Let's keep this method since listeners are optional.  We should not require to pass a empty list everywhere.

Reply:
done